pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -361,26 +295,23 @@ protected void initJCommander() {
         // Automatically generate documents for pulsar-admin
         commandMap.put("documents", CmdGenerateDocument.class);
         // To remain backwards compatibility for "source" and "sink" commands
-        // TODO eventually remove this
-        commandMap.put("source", CmdSources.class);
```

Comment:
can we remove them directly?

Reply:
`source` alias is added to `CmdSources.class`.